infrastructure/src/main/java/org/corfudb/infrastructure/FailureHandlerDispatcher.java

```diff
@@ -70,7 +64,18 @@ public boolean recoverCluster(ServerContext serverContext, Layout recoveryLayout
             corfuRuntime.invalidateLayout();
             if (corfuRuntime.getLayoutView().getLayout().equals(recoveryLayout)) {
                 log.info("Layout Recovered = {}", recoveryLayout);
+            } else {
+                log.warn("Layout recovered with a different layout = {}",
+                        corfuRuntime.getLayoutView().getLayout());
             }
+
+            //TODO: Since sequencer reset is moved after paxos. Make sure the runtime has the latest
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)